docs/markdown/snippets/add_c17_and_c18_standards.md

```diff
@@ -0,0 +1,9 @@
+---
+short-description: Add 'c17' and 'c18' values for c_std option
+...
+
+# Added c17 and c18 as c_std values for recent GCC and Clang Versions
```

Comment:
This needs to be
```markdown
## Added c17 and c18 as c_std values for recent GCC and Clang Versions
```

Note the `##` instead of `#`

That should fix all of your test failures (I think).

Reply:
Sorry about that.  Just made the fix in my branch jrl64:add_c17_to_c_std. Thank you for tracking that down. I never would have found it.